docs/src/main/asciidoc/doc-concepts.adoc

```diff
@@ -3,7 +3,7 @@ This document is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
-[id="concepts-quarkus-documentation"]
+[id="doc-concepts"]
```

Comment:
For consistency, should the ID + filename of this topic be `doc-contribute-concept`? i.e include a middle part that identifies the specific topic area.

Reply:
The concepts is less about "how to contribute" documentation, and the philosophy/concept of how docs should evolve. `doc-concept.adoc` goes with `doc-reference.adoc`...